### PR TITLE
Testcafe - Applicant Profile - My Skills - Fix breaking test.

### PR DESCRIPTION
**Pull request by yonikid15**
### Notes:
- The 'Applicant Profile - My Skills' browser test was breaking for me on travis. It was passing locally thought. I saw it breaking on #2499 as well so I made this change which seems to fix it.

### CODE REVIEW

**Comment by Khristophor** [comment]
Is this failing locally? I'm not seeing failures on other PR's

**Comment by yonikid15** [comment]
> Is this failing locally? I'm not seeing failures on other PR's

No it passed locally for me. I saw it failing on #2499 as well.

File: tests/Browser/2-critical.testcafe.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ test("Applicant Profile - My Skills", async t => {
     .expect(Selector("h3").withText("Writing my application").visible)
     .ok()
     .click(Selector("button").withText("Got it!"))
-    .wait(300)
+    .wait(1234)
```
Comment (Khristophor) [inline tests/Browser/2-critical.testcafe.js]: I'd try changing it to a smaller value first, to see if it fixes the issue (500 maybe). And change the upstream to the PR with the test failing (#2499).

Comment (yonikid15) [reply tests/Browser/2-critical.testcafe.js]: I just made a new PR off the latest dev #2514 which was breaking as well. So my thought was to make the change to the dev branch. But yes I can try making the value smaller.

Comment (yonikid15) [reply tests/Browser/2-critical.testcafe.js]: Looks like lower time works 👍

**Pull request merged**